php/meterpreter/meterpreter.php

```diff
@@ -630,8 +655,16 @@ function rand_xor_byte() {
     return chr(mt_rand(1, 255));
```

Comment:
Good question, I assumed we already did, though the search appears to be unfruitful!